### DISCUSSION TITLE
Feature request regarding Custom Recipies

### DISCUSSION THREAD

**Discussion by ColaIan** [Zrips/CMI#2160] [Feature Requests]
**Description of feature/improvement request:**
1 - Allow the creation of custom recipes containing multiple items in a slot (like 9 stacks of wheat for a compressed wheat item)
2 - Add custom recipes for furnaces, anvils and various machines added in 1.14 (i.e. Blast Furnace, Smoker, Campfire, Stonecutter and Grindstone)
3 - Add the ability to add compostable items for composter

---
### Detailed explanation of what it should do, when and in what situation: ###
1 - Allow more freedom to making recipes
2 - Allow the making of certain custom items in certain machines
3 - Allow more items to be used in the composter

### Basic explanation as to why this feature is needed and/or useful to have: ###
I think these suggestions mentioned above can improve the fun of the gameplay and can let server owners to spend their creativity to improve their servers with a large customizability, thank you!

**Comment by agentlinz** [comment]
I'd like to second this suggestion and add another request regarding it.

- Add the ability to require permission nodes for customrecipes, if desired.
